Resources/doc/menu_options.rst

```diff
@@ -0,0 +1,45 @@
+Setting Default Menu Options
+============================
+
+When creating a menu, menu options are set by passing them to ``addChild()``.
+When all menu items need the same option, adding the same options to each child
+can be cumbersome.
+
+In these cases, you can configure the menu options in the configuration file:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+        knp_menu:
+            default_menu_options:
+                class: menu-item
```

Comment:
there is no ``class`` option in KnpMenu. Please use a real use case in the documentation (which will also help you see that the current implementation is not very usage)